pkg/controller/account/account_controller.go

```diff
@@ -145,11 +148,25 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
+	// Determine if in fedramp env
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		log.Error(err, "Failed retrieving configmap")
+	}
+	ifFedramp, err := controllerutils.IsFedramp(configMap)
```

Comment:
```suggestion
	isFedramp, err := controllerutils.IsFedramp(configMap)
```

Reply:
@NautiluX i think b0755f238ec7e5ac0feea39ba9bbf665baa8e4e4 should address this

i'm not sure i like the "only check at operator startup" model, since to update the value you'd have to restart the operator. i'd much rather it be able to adjust to a change in the configmap while running, even if that means a few more cycles are spent checking the configmap

what do you think?